api/v1alpha08/sonataflowplatform_types.go

```diff
@@ -46,6 +46,57 @@ type SonataFlowPlatformSpec struct {
 	// `sonataflow.org/profile: prod`
 	// +optional
 	Services ServicesPlatformSpec `json:"services,omitempty"`
+	// Persistence defines the platform persistence configuration. When this field is set,
+	// the configuration is used as the persistence for platform services and sonataflow instances
+	// that don't provide one of their own.
+	// +optional
+	Persistence *PlatformPersistenceSpec `json:"persistence,omitempty"`
+}
+
+// PlatformPersistenceSpec configures the DataBase support for both platform services and workflows. For services, it allows
+// configuring a generic database connectivity if the service does not come with its own configured. In case of the workflow
+// the configuration is defined by the image name of the supported database. When the workflow is configured to require
```

Comment:
I think you inverted this description. The service has a DB type connectivity set to it. The workflow supports a generic database config.